.github/workflows/tagbot.yml

```diff
@@ -1,9 +1,25 @@
 name: TagBot
 on:
-  issue_comment:  # THIS BIT IS NEW
+  issue_comment:
     types:
       - created
   workflow_dispatch:
+    inputs:
+      lookback:
+        default: 3
```

Comment:
does this mean it looks backs 3 commits and tries to tag all of them (if not already)?

Reply:
To be perfectly honest, I'm not sure; I just took that from the [TagBot repository README](https://github.com/JuliaRegistries/TagBot/blob/master/README.md).